Aztec/Classes/Private/Libxml2/Data/LeafNode.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+/// Represents any node that can be a leaf in the DOM tree.
+///
+protocol LeafNode {
+    
+    /// Returns the text representation of the node.
+    ///
+    func text() -> String
```

Comment:
I'm not a big fan of properties in protocols for read-access only... I feel like they make the most sense when write access is needed as well.

In this case, considering we only need read access, I find this:

```swift
func text() -> String
```

a bit clearer than this:

```swift
var text: String { get }
```

Does that make sense?